src/enums.ts

```diff
@@ -7,3 +7,10 @@ export enum AggregatorStrategy {
 	AVERAGE = 'average',
 	OMIT = 'omit',
 }
+
+export enum Kind {
```

Comment:
I think it makes more sense to have this as
```ts
export type Kind = keyof MetricsMap
````
as it will automatically be kept up to date and reduce maintenance burden

Reply:
Good call